### DISCUSSION TITLE
Missing management options when in "All entries" view

### DISCUSSION THREAD

**Discussion by MattWilcox** [craftcms/cms#11685] [Ideas]
### Description

Just had a client confused why "today" they didn't have the option to duplicate pages in the CMS like they did yesterday. Turns out that yesterday they were in `Entries > Section Title` and using the checkboxes and cog to duplicate, but today they're in `Entries > All Entries` and trying to do it... and those two views show different options in the cog dropdown.

<img width="1040" alt="Screenshot 2022-07-21 at 11 06 44" src="https://user-images.githubusercontent.com/814039/180189313-e95dd0b8-3ff4-442d-88f9-adcb45bcb36f.png">

vs the same page attempting to be duplicated

<img width="1033" alt="Screenshot 2022-07-21 at 11 06 16" src="https://user-images.githubusercontent.com/814039/180189302-a49380d9-1514-46d8-a887-99caa9059cdd.png">

I've told them how to work around this, but it does seem like a strange difference just because the view context is a little different.

I imagine this is a design choice, but can't seem to mark this as "not a bug". Is it possible for the cog to be a bit smarter and note the context of the selected items to know what options are valid?

### Craft CMS version

3.7.48

### Installed plugins and versions

-

**Comment by brandonkelly** [comment]
> Is it possible for the cog to be a bit smarter and note the context of the selected items to know what options are valid?

To a degree yes; though it requires some rewiring. Definitely not a bug so I’ve moved this to a discussion.

**Reply by MattWilcox** [reply]
Cheers Brandon. Still not used to the Discussion workflow on GitHub, my apologies!
